bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java

```diff
@@ -155,6 +156,9 @@ public void packetReceived(BasePacket packet) {
         ERP1Message msg = (ERP1Message) packet;
 
         EEPType localReceivingType = receivingEEPTypes.get(msg.getRORG());
+        if (Objects.isNull(localReceivingType)) {
```

Comment:
```suggestion
        if (localReceivingType == null) {
```

Reply:
The same but different?

Reply:
Yes, same logic, but that would introduce a compiler warning because of missing annotations for `Hashtable.get`. This is what @lovery wanted to fix in the first place, see https://github.com/openhab/openhab-addons/pull/17531#discussion_r1796912904.

Reply:
Maybe got on the wrong foot as the PR is suppose to fix a NPE and I don’t see where the fix is.

Reply:
So this return is the actual fix I guss?

Reply:
Yes, this reverts the regression from #17531 where the check and return were removed.